app/jobs/checksum_report_job.rb

```diff
@@ -26,31 +26,31 @@ def perform(bulk_action_id, params)
         csv_report = Preservation::Client.objects.checksums(druids: pids)
         File.open(report_filename, 'w') { |file| file.write(csv_report) }
         bulk_action.update(druid_count_success: pids.length) # this whole job is run in one call, so it either all succeeds or fails
-      rescue Preservation::Client::NotFoundError, Preservation::Client::UnexpectedResponseError => e
-        bulk_action.update(druid_count_fail: pids.length)
-        message = "#{Time.current} ChecksumReportJob got error from Preservation Catalog API: #{e.class} #{e.message}"
-        log.puts(message)
-        # honeybadger and other notifications should happen at prescat level, but not getting good message
-        raise message
-      rescue Preservation::Client::ConnectionFailedError => e
-        bulk_action.update(druid_count_fail: pids.length)
-        message = "#{Time.current} ChecksumReportJob failed on call to Preservation Catalog API: #{e.class} #{e.message}"
-        log.puts(message)
-        # if we couldn't connect to preservation API, we have an issue
-        raise message
       rescue StandardError => e
         bulk_action.update(druid_count_fail: pids.length)
-        message = "#{Time.current} ChecksumReportJob creation failed #{e.class} #{e.message}"
-        log.puts(message)
-        # if we couldn't write out the file, we have an issue
-        raise message
+        message = exception_message_for(e)
+        log.puts(message) # this one goes to the user via the bulk action log
+        Rails.logger.error(message) # this is for later archaeological digs
```

Comment:
This is a change in behavior introduced in this branch: instead of raising the error, log it and notify Honeybadger.